safir/src/safir/uws/_constants.py

```diff
@@ -4,13 +4,11 @@
 
 from datetime import timedelta
 
-from arq.cron import Options
-
 __all__ = [
     "JOB_RESULT_TIMEOUT",
     "JOB_STOP_TIMEOUT",
     "UWS_DATABASE_TIMEOUT",
```

Comment:
Should we call this something else now that this isn't really a database worker?

Reply:
I think the database worker terminology is still mostly accurate, since that arq worker is responsible for storing results in the database and the timeout is for the worker that does that. It now does this via Wobbly, which exposes an HTTPS API to the database, but that's more of a protocol change than a functional change.